src/components/DocumentationTopic/ContentNode.vue

```diff
@@ -75,6 +75,17 @@ $docs-code-listing-border-width: 1px !default;
     max-width: 100%;
   }
 
+  // if the image is inside a figure, and caption is after it, remove the bottom margin
```

Comment:
I know you were thinking there might be a better way of writing these styles, but it works for now and we can always revisit it later to refactor. I don't have a better solution that I could come up with quickly. 😅